plugins/plugin-babel/plugin.js

```diff
@@ -43,6 +43,8 @@ module.exports = function plugin(snowpackConfig, options = {}) {
       }
       return {
         '.js': {
+          contents: code,
+          // QUESTION: Do I need this?
           code,
```

Comment:
if this is considered a public API, I'd do the following

```suggestion
          contents: code,
          get code () {
            console.warn('[snowpack] `.code` is deprecated, use `.contents`')
            return code;
          },
```

That will help users to migrate to the new API, and as part of the next breaking release we can simply remove the previously deprecated APIs.

Reply:
Okay, I wanted to get the team's opinions on how to handle the deprecation and how we wanted to let the users know.

Reply:
One idea posed here (https://github.com/pikapkg/snowpack/discussions/1018): what if we simply ignored all binary files and didn’t pass them to plugins (click in for further explanation)? We could still change to `contents` instead here, but it would essentially always be a string, at least as far as plugins are concerned.